lib/index.js

```diff
@@ -75,6 +106,7 @@ function getFooterNode() {
  * @param {Boolean} [options.routing=false] Specify whether to request additional metadata about the recommended navigation destination corresponding to the feature or not. Only applicable for address features.
  * @param {String} [options.worldview="us"] Filter results to geographic features whose characteristics are defined differently by audiences belonging to various regional, cultural, or political groups.
  * @param {Boolean} [options.enableGeolocation=false] If `true` enable user geolocation feature.
+ * @param {'v5'|'v6'} [options.version='v5'] If `v6` use v6 endpoint instead of v5. The default value is set to v5 API.
```

Comment:
Do we want to make the default v6 going forward?

Reply:
I would not use it as default value because it breaks backward compatibility. Because v6 contact is very different from v5 one, every user that expects to receive v5 suggestion will get their code broken